packages/apps-config/src/ui/logos/index.ts

```diff
@@ -239,10 +242,13 @@ export const nodeLogos: Record<string, unknown> = [
   ['darwinia', nodeDarwinia],
   ['darwinia parachain', nodeDarwinia],
   ['Darwinia Runtime Module Library', nodeDarwinia],
-  ['DataHighway', nodeDataHighway],
-  ['DataHighwayChain', nodeWestlake],
-  ['DataHighway Node', nodeDataHighway],
-  ['DataHighway Parachain Collator', nodeDataHighway],
+  ['DataHighway', nodeDataHighwayDefault], // TODO - deprecated?
```

Comment:
we should remove these lines that i've labelled `// TODO - deprecated?` if they're not used anymore